app/src/main/java/com/example/marvelapp/data/service/response/MarvelBaseResponse.kt

```diff
@@ -0,0 +1,5 @@
+package com.example.marvelapp.data.service.response
+
+class MarvelBaseResponse(
+        var data: MarvelResponse
```

Comment:
format the code properly pls

Reply:
![image](https://user-images.githubusercontent.com/73836200/109208680-61cd4700-7789-11eb-8aab-606ae510696f.png)
the same as the comment below